pkg/scrape/scrape.go

```diff
@@ -88,8 +88,8 @@ func newScrapePool(cfg *config.Config, p ingestion.Ingester, logger logrus.Field
 		activeTargets: make(map[uint64]*Target),
 		loops:         make(map[uint64]*scrapeLoop),
 
-		metrics:     m,
-		poolMetrics: m.poolMetrics(cfg.JobName),
+		metrics:                m,
+		poolMetrics:            m.poolMetrics(cfg.JobName),
```

Comment:
A nit: it seems the previous version conforms `go fmt`, so let's not change it

Reply:
Thanks @kolesnikovae  I have updated the pr.